modules/exploits/multi/local/burp_extension_persistence.rb

```diff
@@ -0,0 +1,411 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::File
+  include Msf::Post::Unix # whoami
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::FileDropper
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Post::Windows::Registry
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Burp Extension Persistence',
+        'Description' => %q{
+          This module adds a java based malicious extension to the Burp Suite configuration file.
+          When burp is opened, the extension will be loaded and the payload will be executed.
+
+          Tested against Burp Suite Community Edition v2024.9.4 on Kali.
+          Tested against Burp Suite Professional ??? on Kali.
+          Tested against Burp Suite Community Edition v2024.10.3 on Windows 10.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'h00die', # Module
+        ],
+        'DisclosureDate' => '2025-01-01',
+        'SessionTypes' => [ 'shell', 'meterpreter' ],
+        'Privileged' => false,
+        'References' => [
+          [ 'URL', 'https://portswigger.net/burp/documentation/desktop/extensions/creating' ],
+          [ 'URL', 'https://portswigger.net/burp/documentation/desktop/troubleshooting/launch-from-command-line' ]
+        ],
+        'DefaultOptions' => {
+          # 25hrs, you know, just in case the user doesn't open Burp for a while
+          'WfsDelay' => 90_000,
```

Comment:
I think `25*60*60` would be a tad clearer

Reply:
can/will be addressed in the persistence PR